src/main.rs

```diff
@@ -193,7 +192,7 @@ fn render_axis(
     // Only want to load this once
     thread_local! {
         /// This is an example for using doc comment attributes
```

Comment:
lmao. I am embarrassed. I left in a comment from some code I copied off the internet. Please delete this. :sweat_smile: :joy: